README.rst

```diff
@@ -52,17 +52,34 @@
 Welcome to sktime
 =================
 
-  A unified framework for machine learning with time series
+The unified framework for machine learning with time series. Our vision:
```

Comment:
I still prefer "A" - don't think we need to claim being the "special" one in whatever sense that may be.

Reply:
I have preference for 'The'. But it is not strong preference. Either would be fine

Reply:
I prefer "A", "The" seems a bit too much to claim

Reply:
ok, fine with me

Reply:
what about "the scikit-learn-like"? that is more like a definition